src/styles/_global.scss

```
@@ -7,6 +7,14 @@ $cadet-color-3: #34495e;
 $cadet-color-4: #ced9e0;
 $cadet-color-5: #ffffff;
 
+:root {
+  --cadet-color-1: $cadet-color-1;
```

Stylelint (17.12.0)

[error] Expected variable $cadet-color-1 to be interpolated when using it with --cadet-color-1 (scss/dollar-variable-no-missing-interpolation)

(scss/dollar-variable-no-missing-interpolation)